easypost/util.py

```diff
@@ -0,0 +1,36 @@
+from typing import List
+
+from easypost.easypost_object import EasyPostObject
+from easypost.error import Error
+
+
+class Util:
```

Comment:
nit: we don't need a Util class. This can just be an assortment of functions in a `util.py` file that we import individually as needed. The way it's configured now, every function in `Util` will be available everywhere we import `Util`

Reply:
Good point. Flattened the module.